canonical/ubuntu.com
# feat: /azure/pro redesign

## Done

- Applied redesign as per [doc](https://docs.google.com/document/d/1WaQ5n3GxwZzbX0T6rLiRNejFvxIPIHd5V5XgdAwKmSw/edit) and [design](https://www.figma.com/design/MinLlXouBnEDqaoAL3bTWC/24.10-ubuntu.com%2Fazure?node-id=282-1394&node-type=frame&m=dev)

## QA

- View the site locally in your web browser at: https://ubuntu-com-14315.demos.haus/azure/pro
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-12829
